packages/cli/tests/e2e/apiconnector/verifyScaffold.tests.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import path from "path";
+import "mocha";
+import { expect } from "chai";
+import { getTestFolder, getUniqueAppName, cleanUp } from "../commonUtils";
+import { CliHelper } from "../../commonlib/cliHelper";
+import { Capability } from "../../commonlib/constants";
+import * as fs from "fs-extra";
+
+describe("Add api-connection cli", () => {
+  const testFolder = getTestFolder();
+  const appName = getUniqueAppName();
+  const projectPath = path.resolve(testFolder, appName);
+  before(async () => {});
+  after(async () => {
+    await cleanUp(appName, projectPath, false, false, false);
+  });
+
+  it("Generate Sample codes success - basic auth", async () => {
+    await CliHelper.createProjectWithCapability(appName, testFolder, Capability.Bot);
+    await CliHelper.addExistingApi(
+      projectPath,
+      `--api-connector-auth-type basic --component-select bot --api-connector-user-name basictest --api-connector-endpoint https://localhost.basictest.com --api-connector-name basictest --interactive false`
```

Comment:
is it possible to remove the `--api-connector` prefix in parameters? we can have another PR to do this

Reply:
the `--component-select` can also be renamed to `--component`

Reply:
the component change to component

Reply:
remove api-connector prefix will do later PR